src/main/java/duke/Message.java

```diff
@@ -0,0 +1,36 @@
+package duke;
+
+/**
+ * Represents the message with appropriate flags needed by the display.
+ */
+public class Message {
+    private boolean isExit;
```

Comment:
You could consider using shouldExit. However, I do think isExit is also a very reasonable boolean name.